public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -228,7 +232,11 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       return;
     }
     this.resetSelectedGraphs();
-    this.showGenderGraph = true;
+    if (this.ehrDomain.name === 'Measurements') {
+      this.showMeasurementGenderBins = true;
```

Comment:
Why is this case treated differently? Comment?

Reply:
Done.